genai-perf/tests/test_data_parser/test_llm_profile_data_parser.py

```diff
@@ -210,7 +210,7 @@ def test_triton_llm_profile_data(
             - experiment 2: [3, 4]
         """
         config = ConfigCommand({"model_name": "test_model"})
-        config.tokenizer.name = DEFAULT_TOKENIZER
+        config.tokenizer.name = "hf-internal-testing/llama-tokenizer"
```

Comment:
This feels like it opens us to subtle errors.
Why did we move away from using a variable for the default tokenizer?

Reply:
Multiple users were upset because the default tokenizer was set when no tokenizer was provided, resulting in the wrong numbers. It made more sense to require a tokenizer to always be set, unless it could be inferred from the model name (usually can be). So we avoid silent errors and improve the user experience.

It's only a very niche case where a user does not care about the numbers and therefore does not want to set the tokenizer, e.g. for testing or validation. If we want to declare a DEFAULT_TESTING_TOKENIZER just so that our tests all have the same tokenizer, we can do so, though I'd rather do that in a separate PR. I think the marginal utility would be negligible.

Reply:
I forgot about that case as well.
In light of that, this change makes way more sense.